### ISSUE TITLE
play sound when file copy/move completes

### ISSUE THREAD

**Issue by stefanocaselli** [tablacus/TablacusExplorer#67]
Hello,
is it possible to play a sound after long file operations are completed? (like copy, move or delete files)
For "long" I'm thinking about > 5 seconds (configurable by the user).
Thanks for this great file manager.

**Comment by tablacus** [comment]
I think I can do it if I improve an add-on "Multi Process".
Thank you,

**Comment by stefanocaselli** [comment]
It would be wonderful (I'm already using Multi Process plugin)
Thanks.


2016-10-29 14:17 GMT+02:00 Gaku <notifications@github.com>:

**Comment by tablacus** [comment]
I've update an add-on "Multi Process".
![2](https://cloud.githubusercontent.com/assets/5156977/19930600/89d61320-a14c-11e6-865d-b7a9ce3bdfe9.png)
Audio file: mp3, wav, mp4, wma, etc...
Best regards,

**Comment by stefanocaselli** [comment]
Hello,
thank you for the update, but it doesn't play anything for me (tested using
2 different pc running win10 x64).
also I find that the "multi process" isn't working as expected: the
tablacus gui is blocked for the entire duration of the copy process.
how can I help you to debug this feature?
thanks.


2016-11-02 14:36 GMT+01:00 Gaku <notifications@github.com>:

**Comment by tablacus** [comment]
Add-ons "Multi Process", "Fire File Copy", "Fast Copy" and "Multi Thread" are conflicts.
Use either.

**Comment by stefanocaselli** [comment]
I only use Multi Process:

[image: Immagine incorporata 1]



2016-11-02 23:30 GMT+01:00 Gaku <notifications@github.com>:

**Comment by tablacus** [comment]
When you selected the media file, do you see this bar?
![7](https://cloud.githubusercontent.com/assets/5156977/19966167/f1991766-a20d-11e6-84a3-96bbeca8057f.png)
Best regards,

**Comment by stefanocaselli** [comment]
Yes, I see the bar, and I also pressed the play button to listen to the
media I have chosen (the same ding.wav file in your screenshot).
Maybe it's windows 10... I will try it on windows 7, when I can.
Thanks.


2016-11-03 13:40 GMT+01:00 Gaku <notifications@github.com>:

**Comment by tablacus** [comment]
Oh, this PC is Windows 10.
Take "Multi Process" to the top for now.
![8](https://cloud.githubusercontent.com/assets/5156977/20011968/cd341ed2-a2f0-11e6-84eb-31e0ee472d84.png)
Check radio button for "Multi Process", and push "Up".
Or Drag and drop.

-----

If it doesn't work, would you tell me the file operations that you did.
Best regards,

**Comment by stefanocaselli** [comment]
Hi,
I was testing by copying a file in the same directory. That way it doesn't
play the sound.
If I copy into different directories then it works.
Thanks for your support.


2016-11-04 16:53 GMT+01:00 Gaku <notifications@github.com>:

**Comment by tablacus** [comment]
All right.
Thank you for your information.
